container/kubernetes/onboarding/gatewaydeployment.yaml

```diff
@@ -0,0 +1,140 @@
+version: 1 
+namespace: oisp-devices
+CreateDeployments:
+  - ApplicationName: gasversorung
+    NodeName: shailesh-nuc
+    ToolVersion: 1
+    DeploymentConfig:
+       URL: "https://github.com/shailesh83/Gateway-Deployment"
+       Directory:  "deploy-gas"
+       GitTag: "650541f301347d469297b9cfcddf39b3dce52370"
+    MachineConfig:
+       URL: "https://github.com/mattmikulina/IndustryFusion-machine-configs/"
```

Comment:
Same comment here - i dont't think we should put those concrete urls and tags into the public - use placeholder. And keep the original in a private repo.

Reply:
Should i copy Machine Config to my public repo and use it here.

Reply:
yes or copy it to the IndustryFusion-machine-configs - with approval from Konstantin

Reply:
ok